README.md

```diff
@@ -3,79 +3,49 @@
 
-## Overview
+  - **init** - Initialize a Polymer project from custom templates
+  - **install** - Install bower dependencies as well as [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) for testing
+  - **serve**	- Serve elements and applications locally
+  - **lint** - Lint a project to catch errors before your users
+  - **test** - Test your project with [`web-component-tester`](https://github.com/Polymer/web-component-tester/)
+  - **build**	- Build an application optimized for production
+  - **analyze** - Generate an analyzed JSON representation of your element or application
 
-Polymer-CLI includes a number of tools for working with Polymer and Web Components:
+> **For a detailed overview of the CLI, how it works and when to use it, check out the official
+[Polymer CLI guide](https://www.polymer-project.org/2.0/docs/tools/polymer-cli).**
+> This README will focus on the individual CLI commands and how to run them.
 
-  * __init__ - Initializes a Polymer project from one of several templates
-  * __build__	- Builds an application-style project
-  * __lint__ - Lints the project
-  * __serve__	- Runs a development server
-  * __test__ - Runs tests with web-component-tester
 
 ## Installation
 
-Install via npm:
-
-    $ npm install -g polymer-cli
-
-Then run via `polymer <command>`:
-
-    $ polymer help
-
-## Project Structure
-
-Polymer-CLI is somewhat opinionated about project structure.
-
-There are two type of projects:
-
-* Elements projects
-
-  Element projects contain one or more reusable element definitions, intended to be used from other elements, applications or pages. Element definitions live at the top level of the project so they are easy to import. Elements import their dependencies with relative paths that reference sibling folders to the project folder.
-
-* Application projects
-
-  Application projects are self-contained and intended to be deployed as a standalone website. Application projects contain elements in a `src/` folder and import their dependencies with absolute paths, or relative paths that reference folders inside the project folder.
-
-### Application Styles
-
-Polymer-CLI currently supports two styles of applications:
-
-  * Monolithic applications, which have a single entrypoint (usually index.html) and eagerly import all dependencies.
-
-  * "App shell" applications, which have a very lightweight entrypoint, an app-shell with startup and routing logic, and possibly lazy loaded fragments.
-
-### App-shell Structure
-
-App-shell apps are currently the preferred style for Polymer CLI, and most commands are being optimized to support them. App-shell apps usually have client-side routing (see the [app-route](https://github.com/PolymerElements/app-route) element), and lazy load parts of the UI on demand.
-
-Polymer-CLI supports this style by understand these different types of files:
+```bash
+$ npm install -g polymer-cli
+# or...
+$ yarn add global polymer-cli
```

Comment:
Should we put yarn first, as it's such a significant installation performance win?